stubs/ext-ds.php

```diff
@@ -462,37 +598,80 @@ public function copy(): Pair
  */
 interface Sequence extends Collection, ArrayAccess
 {
+    /**
+     * @return Sequence<TKey, TValue>
+     * @psalm-mutation-free
+     */
+    public function copy(): Sequence;
+
+    /**
+     * @return array<TKey, TValue>
+     * @psalm-mutation-free
+     */
+    public function toArray(): array;
```

Comment:
@muglug The failing tests are not related in my opinion. However I found a different error - TKey type doesn't exists in Sequence. This tells me that there is no test that the stubs are valid which I think there should be.

```suggestion
    /**
     * @return Sequence<int, TValue>
     * @psalm-mutation-free
     */
    public function copy(): Sequence;

    /**
     * @return array<int, TValue>
     * @psalm-mutation-free
     */
    public function toArray(): array;
```